plugins/module_utils/network/sonic/argspec/bgp_af/bgp_af.py

```diff
@@ -50,6 +50,20 @@ def __init__(self, **kwargs):
                                 'advertise_pip_ip': {'type': 'str'},
                                 'advertise_pip_peer_ip': {'type': 'str'},
                                 'advertise_svi_ip': {'type': 'bool'},
+                                'route_advertise_list': {
+                                    'elements': 'dict',
+                                    'options': {
+                                        'advertise_afi_safi': {
+                                            'choices': ['ipv4', 'ipv6'],
```

Comment:
Please change "advertise_afi_safi" to "advertise_afi". (See the related comment in the "plugins/module_utils/network/sonic/config/bgp_af/bgp_af.py" file.)